frontend/ui/lib/widgets/trip_view_widget.dart

```diff
@@ -14,26 +14,30 @@ class _TripWidgetState extends State<TripViewWidget> {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-          border: Border.all(
-            color: Colors.black,
+    return CustomScrollView(
+      slivers: [
+        SliverAppBar(
```

Comment:
Nit: It's a little weird to have a sliver app bar and another top-level app-bar (both with stack navigation buttons as well). No need to fix it here, but we should consider the UI experience.